chroma-manager/requirements.dev

```diff
@@ -6,10 +6,10 @@
 # http://wiki.whamcloud.com/display/HYD/Adding+and+Updating+IML+Requirements
 behave>=1.1.0
 django-extensions
-django-nose
+django-nose==1.2
 django-supervisor==0.3.0
 fudge
-log4tailer
+https://bitbucket.org/jordilin/alerta/downloads/log4tailer-3.0.9.tar.gz
```

Comment:
Bitbucket is a GitHub-like/alternative code hosting service.

There is no good reason that I can think of for us to fork that and then have to maintain it just so that it's on GitHub instead of Bitbucket.